docs/coding-style.md

```diff
@@ -5,7 +5,7 @@ Our code style guidelines are based on the [Android Code Style Guidelines for Co
 * Line length is 120 characters
 * FIXME must not be committed in the repository use TODO instead. FIXME can be used in your own local repository only.
 
-On top of the Android linter rules (best run for this project using `./gradlew lintVanillaRelease`), we use two linters: [Checkstyle](http://checkstyle.sourceforge.net/) (for Java and some language-independent custom project rules), and [ktlint](https://github.com/pinterest/ktlint) (for Kotlin).
+On top of the Android linter rules (best run for this project using `./gradlew lintWordPressVanillaRelease`), we use two linters: [Checkstyle](http://checkstyle.sourceforge.net/) (for Java and some language-independent custom project rules), and [ktlint](https://github.com/pinterest/ktlint) (for Kotlin).
```

Comment:
👋 @AliSoftware !

Thanks for the comment. 🙏 

`Detekt` is already being mentioned at the end of this document in its own separate section. That work was done way back when `Detekt` got integrated into the codebase.

Is that what you wanted?

Reply:
Ah yes perfect! Missed it because I only focused on the diff without looking at the whole file (and missed parts already there that this PR didn't change), but that paragraph LGTM 👍

Reply:
Coolio, then we are on the same page, again, thanks for the review! 👍

Reply:
You're welcome 🙂